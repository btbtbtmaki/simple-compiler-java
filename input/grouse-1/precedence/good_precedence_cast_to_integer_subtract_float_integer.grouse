main {

  print 2.8 :int - 6 nl;
  print 7.9 :int - 52 nl;
  print 8.1 :int - 55 nl;
  print 7.4 :int - 87 nl;
  print 3.2 :int - 59 nl;
  print 4.8 :int - 67 nl;
  print 6.1 :int - 24 nl;
  print 4.3 :int - 55 nl;
  print 5.1 :int - 75 nl;
  print 9.1 :int - 39 nl;
  print 2.4 :int - 91 nl;
  print 6.2 :int - 15 nl;

  print 26 - 9.1 :int nl;
  print 3 - 5.3 :int nl;
  print 67 - 2.5 :int nl;
  print 7 - 6.6 :int nl;
  print 34 - 1.7 :int nl;
  print 36 - 6.9 :int nl;
  print 45 - 2.5 :int nl;
  print 45 - 4.5 :int nl;
  print 1 - 3.9 :int nl;
  print 22 - 6.7 :int nl;
  print 3 - 8.2 :int nl;
  print 47 - 4.6 :int nl;

  print 2.5 :int - 4.0 :int nl;
  print 4.2 :int - 2.5 :int nl;
  print 8.2 :int - 7.8 :int nl;
  print 5.1 :int - 5.2 :int nl;
  print 7.1 :int - 3.0 :int nl;
  print 6.2 :int - 8.0 :int nl;
  print 2.2 :int - 8.3 :int nl;
  print 6.0 :int - 1.1 :int nl;
  print 8.5 :int - 8.3 :int nl;
  print 8.5 :int - 3.2 :int nl;
  print 2.8 :int - 1.0 :int nl;
  print 3.1 :int - 2.8 :int nl;

}

