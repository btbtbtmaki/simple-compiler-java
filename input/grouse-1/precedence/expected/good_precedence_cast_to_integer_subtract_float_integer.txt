-4
-45
-47
-80
-56
-63
-18
-51
-70
-30
-89
-9
17
-2
65
1
33
30
43
41
-2
16
-5
43
-2
2
1
0
4
-2
-6
5
0
5
1
1
